sidechain/consensus/aura/src/block_importer.rs

```diff
@@ -89,15 +91,16 @@ impl<
 	> where
 	Authority: Pair,
 	Authority::Public: std::fmt::Debug,
-	PB: ParentchainBlockTrait<Hash = H256>,
-	SB: SignedBlockT<Public = Authority::Public> + 'static,
-	SB::Block: BlockT<ShardIdentifier = H256>,
+	ParentchainBlock: ParentchainBlockTrait<Hash = H256>,
+	SidechainBlock: SignedBlockTrait<Public = Authority::Public> + 'static,
```

Comment:
This type should probably be named `SignedSidechainBlock`? It has confused me a couple of times, because we sometimes use `SB` for the 'regular' sidechain block, and sometimes for the signed sidechain block.

Reply:
I agree here, confused me too.

Reply:
Adapted :)